db/views/aggregations_search_last_thirty_days_v08.sql

```diff
@@ -0,0 +1,41 @@
+SELECT
+  dimensions_editions.warehouse_item_id as warehouse_item_id,
+  dimensions_editions.title as title,
+  dimensions_editions.document_type as document_type,
+  dimensions_editions.base_path as base_path,
+  dimensions_editions.organisation_id as organisation_id,
+  dimensions_editions.id as dimensions_edition_id,
+  aggregations.upviews::bigint as upviews,
+  aggregations.pviews::bigint as pviews,
+  aggregations.feedex::bigint as feedex,
+  aggregations.useful_yes::bigint as useful_yes,
+  aggregations.useful_no::bigint as useful_no,
+  CURRENT_DATE as updated_at,
```

Comment:
I'm wondering what the space trade off is here, given that this value is being stored for every single entry in this view (and I see this being added to other views as well).

How big are the views already, and how much space does this look to add?

Reply:
I've come up with a couple of possible alternatives...

PostgreSQL apparently can record some information about commit times, https://www.postgresql.org/docs/9.6/functions-info.html#FUNCTIONS-COMMIT-TIMESTAMP which could be useful, but would require some configuration changes, and I'm also uncertain about the space/performance overhead.

Another option would be to have another table which records the last updated time for each view. That would only require as many rows as we have views, and if we update the table in the same transaction as the materialised view, it should be accurate.

Reply:
@cbaines Thanks for the suggestions. The views aren't that big and although a separate table is definitely a cleaner approach I don't feel it's enough of a benefit at the moment. Will definitely look into this if it does.